citi-onboarding/Delta-odonto
# Feature/our clients and blog sections

## What I did
- Made the models for OurBlog, OurClients and ArticleCarousel and its respectives routes.
## How to test
- On terminal you should type de following commands to access the server folder and run the index file:
```
cd server
node index.js
```
- Open a second terminal on client folder and run the following commands to habilite the Keystone interface
```
cd client
yarn server
```

- Go to a browser and access the adress http://localhost:3001/admin/ and you should be able to see the Keystone interface. (Use the password find on server/updates/0.0.1-admin.js in case its needed.)

- You should be able to see OurBlog, OurClients and ArticleCarousel sections and edit them as well.
